mula/scheduler/server/handlers/schedules.py

```diff
@@ -246,3 +254,20 @@ def patch(self, schedule_id: uuid.UUID, schedule: serializers.SchedulePatch) ->
             ) from exc
 
         return updated_schedule
+
+    def delete(self, schedule_id: uuid.UUID) -> None:
+        try:
+            self.ctx.datastores.schedule_store.delete_schedule(schedule_id)
+        except storage.errors.StorageError as exc:
+            raise fastapi.HTTPException(
+                status_code=fastapi.status.HTTP_500_INTERNAL_SERVER_ERROR,
+                detail=f"error occurred while accessing the database [exception: {exc}]",
+            ) from exc
```

Comment:
This seems oddly specific at this point. Is there a reason to expect StorageErrors specifically when deleting? I'd expect the probability of StorageErrors, and the subsequent message to the user to be the same for all endpoints. If so, we'd be better off letting the Exception bubble up to the main loop, and handle it there using a generic error handler or middleware.

Reply:
Referenced this in a different issue, and will be picked up there.

Reply:
https://github.com/minvws/nl-kat-coordination/issues/3162